src/server/session.ts

```diff
@@ -627,15 +839,18 @@ namespace ts.server {
                                 NewLineCharacter: "\n",
                                 ConvertTabsToSpaces: formatOptions.ConvertTabsToSpaces,
                                 IndentStyle: ts.IndentStyle.Smart,
+                                SendMetrics: formatOptions.SendMetrics,

```

Comment:
should not be called formatOptions any longer, should be editorOptions, so is getFormatOptions.